### PR TITLE
Fix grammar 'and you AWS' -> 'and your AWS'

### PR DESCRIPTION
**Pull request by trayo**
**Description of changes:**

Update configuration description to fix grammar 
> If this command is run with no arguments, you will be prompted for configuration values such as your AWS Access Key Id **and you AWS** Secret Access Key.

Should be read as:

> If this command is run with no arguments, you will be prompted for configuration values such as your AWS Access Key Id **and your AWS** Secret Access Key.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kdaily** [comment]
Thanks @trayo!

File: awscli/examples/configure/_description.rst [hunk 1]
```diff
@@ -34,4 +34,4 @@ The following configuration variables are supported in the config file:
 
 For more information on configuration options, see `Configuring the AWS Command Line Interface`_ in the *AWS CLI User Guide*.
 
-.. _`Configuring the AWS Command Line Interface`: http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html
```
Comment (kdaily) [inline awscli/examples/configure/_description.rst:37]: Can you revert this change? doesn't look like it's relevant here.

**Review by kdaily — CHANGES_REQUESTED** [review]
Thanks! Looks good overall, if you can get rid of the inadvertent change to line 37.

File: awscli/examples/configure/_description.rst (hunk 1 above)
Comment (trayo) [reply awscli/examples/configure/_description.rst:37]: Hi @kdaily, I'm not sure what GitHub is showing as being changed here. I tried putting the raw original text into two different diff checkers (https://www.diffchecker.com/ and https://text-compare.com/) and two different online editors that show hidden characters (https://www.soscisurvey.de/tools/view-chars.php and https://dostring.com/show-hidden-characters) and neither show a difference other than changing "you" to "your".

Comment (kdaily) [reply awscli/examples/configure/_description.rst:37]: Thanks for checking, I'm fine w/ leaving it then.

**Review by kdaily — APPROVED** [review]

**Review by stealthycoin — APPROVED** [review]

**Review by kyleknap — APPROVED** [review]
Thanks!
